Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -77,6 +77,26 @@
 CROWDSTRIKE_INDICATOR_RELATION_FIELDS = ['reports', 'actors', 'malware_families', 'vulnerabilities', 'relations']
 
 
+def kill_chain_standard_values(phases: list | None):
```

Comment:
So there were a couple of issues here no? Actually all values (not only c2) were different between CS to XSOAR. 
Also, it seems there are cases where this is a list. How does it aligns with the fact this is a single select value?

Reply:
STIX Kill Chain Phases is multiselect
the field is case insensitive, so only c2 and Actions on Objectives really should have been an issue, but yes none were srictly according to standard